documentation/src/main/markdown/runningdetect/autonomousscan.dita

```diff
@@ -0,0 +1,344 @@
+                    <p>Run <keyword keyref="company_name"/>
+                        <keyword keyref="solution_name"/> in Autonomous mode by providing the
+                            <codeph>--detect.autonomous.scan.enabled=true</codeph> parameter and any
+                        other supported parameters that you require. (See limitations section for
+                        parameters that are not supported in Autonomous mode.)</p>
+                </li>
+                <li>
+                    <p><keyword keyref="company_name"/>
+                        <keyword keyref="solution_name"/> will determine which tools and detectors are
+                        appropriate and available to run, including Package Manager, Signature, and
+                        Binary Scanning.</p>
+                </li>
+                <li>
+                    <p>Scans will include any analyzable content of user specified locations as well
+                        as source or binaries located in the run directory.</p>
+                </li>
+                <li>
+                    <p>Once complete, scan findings can be viewed in the BDIO file produced, or in
+                        the <keyword keyref="blackduck_product_name"/> UI if <keyword keyref="blackduck_product_name"/> has been configured.</p>
+                </li>
+            </ol>
+        </body>
+    </topic>
+    <topic id="subsequent-scan-workflow">
+        <title>Subsequent Scan Workflow</title>
+        <body>
+            <ol>
+                <li>
+                    <p>Run <keyword keyref="company_name"/>
+                        <keyword keyref="solution_name"/> by providing the
+                            <codeph>--detect.autonomous.scan.enabled=true</codeph> parameter.</p>
+                </li>
+                <li>
+                    <p><keyword keyref="company_name"/>
+                        <keyword keyref="solution_name"/> will determine if any user provided arguments
+                        or properties should take precedence over values in the existing scan
+                        settings file, and run the appropriate available tools and detectors.</p>
+                </li>
+                <li>
+                    <p>Once complete, scan findings can be viewed in the BDIO file produced, or in
+                        the <keyword keyref="blackduck_product_name"/> UI if <keyword keyref="blackduck_product_name"/> has been configured.</p>
+                </li>
+            </ol>
+        </body>
+    </topic>
+    <topic id="scan-mode-scenarios">
+        <title>Scan mode scenarios</title>
+        <body>
+            <ul>
+                <li>
+                    <p><xref href="../properties/configuration/blackduck-server.md" format="markdown">ONLINE mode</xref>: With <keyword keyref="blackduck_product_name"/> configuration or proxy configuration
+                        set and <codeph>blackduck.offline.mode</codeph> set to false, or overridden
```

Comment:
We don't need to set blackduck.offline.mode=false explicitly with  blackduck configuration
`and <codeph>blackduck.offline.mode</codeph> set to false` this part can be removed.

Reply:
@sig-mmittal @devmehtasynopsys What if I say "...<codeph>blackduck.offline.mode</codeph> not set to true" ?

Reply:
I made the wording change.